lib/rubocop/config_loader.rb

```diff
@@ -220,6 +222,22 @@ def check_duplication(yaml_code, absolute_path)
         end
       end
 
+      def check_enabled_value(hash, parent = nil)
```

Comment:
I guess we should change the name of this method to something more generic - e.g. `check/verify_cop_config_values` or something like this.

Reply:
Renamed it! :heart_eyes: